src/plugins/ui/sort.ts

```diff
@@ -259,13 +266,17 @@ export class SortPlugin extends UIPlugin {
     // Update in case of merges in the zone
     let cells = this.mainCells(sheetId, zone);
 
-    if (this.hasHeader(cells)) {
+    if (!options.hasNoHeader && this.hasHeader(cells)) {
```

Comment:
shoundl't this option be called  `ignoreHeader`?

Reply:
Not that my name is any good, but wouldn't `ignoreHeader` imply that there is headers, but that we leave them as it ?

Reply:
`ignoreFirstRow` ? :man_shrugging:

Reply:
> Not that my name is any good, but wouldn't `ignoreHeader` imply that there is headers, but that we leave them as it ?

ignoreHeader would imply that even if there are headers (which is determined by the function hasHeader) we would ignore them, which is exactly what you are doing.  I stand by my suggestion ^^ I also feel it is less cryptic than ignoreFirstRow

Reply:
How about `sortHeaders` ?

Reply:
not bad